samples/matter/thermostat/README.rst

```diff
@@ -185,6 +185,16 @@ Building and running
 
 .. include:: /includes/ipc_radio_conf.txt
 
+Building the Matter over Wi-Fi sample variant on nRF5340 DK with nRF7002EK shield
```

Comment:
Don't we need to change pm statics for thermostat as well?

Reply:
Template sample is already updated.